### PR TITLE
Update src/modules/AppCore.js

### PR DESCRIPTION
**Pull request by neighborhood999**
Try update `src/modules/AppCore.js`

### CODE REVIEW

**Comment by EragonJ** [comment]
@neighborhood999 mind to check what's going on in unit test :) ?

**Comment by neighborhood999** [comment]
@EragonJ Sorry, I miss check PR, I already fixed it.

File: src/modules/AppCore.js [hunk 1]
```diff
@@ -1,61 +1,60 @@
...
+
+  getEnvInfo() {
+    const envFilePath = Path.join(App.getAppPath(), 'env.json');
+    const envInfo = Fs.readFileSync(envFilePath, 'utf8');
+    return JSON.parse(envInfo);
+  }
+
+  getPackageInfo() {
+    const packageFilePath = Path.join(App.getAppPath(), 'package.json');
+    const packageInfo = Fs.readFileSync(packageFilePath, 'utf8');
```
Comment (EragonJ) [inline src/modules/AppCore.js]: @neighborhood999 I still prefer to use `const` for constants and required (global) modules. Let's try to use `let` instead next time :)

**Comment by EragonJ** [comment]
Thanks !!!

File: src/modules/AppCore.js (hunk 1 above)
Comment (neighborhood999) [inline src/modules/AppCore.js]: OK, I will.

**Pull request merged**